.github/workflows/skribenten.yml

```diff
@@ -1,4 +1,4 @@
-name: "Build and deploy Skribenten"
+name: "Skribenten backend"
```

Comment:
My 2️⃣ cents, men liker å kalle workflows for "deploy: skribenten-backend" for å skille fra andre workflows som "verify:APP", da sorteres det nice i github også.

Reply:
Denne workflowen kjører også for PR (compile og test), så da funker det ikke helt å ha deploy i navnet uten å ha build

Reply:
Det vi har gjort andre steder er at vi har compile og test i egen "verify: APP" workflow.

Var bare noe jeg kom på siden jeg så du endret navn. Bare en tanke, ikke noe å prioritere nå for å komme ut i prod

Reply:
Ja, tenker vi kan ta en diskusjon på det senere.